.buildkite/scripts/pipeline.sh

```
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
 if [[ "${BUILDKITE_BRANCH}" == "master" ]]; then
-  export CHART_CHANGES=$(git diff --name-only HEAD~1 | sed -rn '/^charts\/[a-zA-Z0-9-]+\/(templates\/.*|crds\/.*|Chart.yaml|values.yaml|values.schema.json|LICENSE|README.md)/{q1}' && echo false || echo true)
+  export CHART_CHANGES=$(git diff --name-only HEAD~1 | sed -rn '/^charts\/[a-zA-Z0-9-]+\/(templates\/.*|crds\/.*|Chart.yaml|values.yaml|values.schema.json|LICENSE|README.md|README.md.gotmpl)/{q1}' && echo false || echo true)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)